doc/releases/changelog-dev.md

```diff
@@ -32,41 +22,99 @@
+
+* The new return system is now activated and public facing. The QNode keyword argument `mode` is replaced by the boolean  
+  `grad_on_execution`.
+  [(#3957)](https://github.com/PennyLaneAI/pennylane/pull/3957)
+  [(#3969)](https://github.com/PennyLaneAI/pennylane/pull/3969)
+
+<h3>Improvements 🛠</h3>
+
+<h4>Next generation device API</h4>
+
+* The `sample_state` function is added to `devices/qubit` that returns a series of samples based on a given
+  state vector and a number of shots.
+  [(#3720)](https://github.com/PennyLaneAI/pennylane/pull/3720)
 
 * The `simulate` function added to `devices/qubit` now supports measuring expectation values of large observables such as
   `qml.Hamiltonian`, `qml.SparseHamiltonian`, `qml.Sum`.
   [(#3759)](https://github.com/PennyLaneAI/pennylane/pull/3759)
 
-* Added a `Shots` class to the `measurements` module to hold shot-related data.
-  [(#3682)](https://github.com/PennyLaneAI/pennylane/pull/3682)
+* The `apply_operation` function added to `devices/qubit` now supports broadcasting.
+  [(#3852)](https://github.com/PennyLaneAI/pennylane/pull/3852)
 
-<h3>Improvements</h3>
+* `pennylane.devices.qubit.preprocess` now allows circuits with non-commuting observables.
+  [(#3857)](https://github.com/PennyLaneAI/pennylane/pull/3857)
+
+* Adjoint differentiation support for the new qubit state-vector device has been added via
+  `adjoint_jacobian` in `devices/qubit`.
+  [(#3790)](https://github.com/PennyLaneAI/pennylane/pull/3790)
+
+* qml.devices.qubit.measure` now computes the expectation values of `Hamiltonian` and `Sum`
+  in a backpropagation-compatible way.
+  [(#3862)](https://github.com/PennyLaneAI/pennylane/pull/3862/)
+
+<h4>Performance improvements</h4>
 
 * Added a new decomposition to `qml.SingleExcitation` that halves the number of
   CNOTs required.
   [(3976)](https://github.com/PennyLaneAI/pennylane/pull/3976)
 
-* The default gaussian device and parameter shift cv support the new return system but only for single measurement.
-  [(3946)](https://github.com/PennyLaneAI/pennylane/pull/3946)
-
 * Improve the efficiency of `tapering()`, `tapering_hf()` and `clifford()`.
   [(3942)](https://github.com/PennyLaneAI/pennylane/pull/3942)
 
 * Update Pauli arithmetic to more efficiently convert to a Hamiltonian.
   [(#3939)](https://github.com/PennyLaneAI/pennylane/pull/3939)
 
-* Keras and Torch NN modules are now compatible with the new return type system.
-  [(#3913)](https://github.com/PennyLaneAI/pennylane/pull/3913)
-  [(#3914)](https://github.com/PennyLaneAI/pennylane/pull/3914)
-
 * The adjoint differentiation method now supports more operations, and does no longer decompose
   some operations that may be differentiated directly. In addition, all new operations with a
   generator are now supported by the method.
   [(#3874)](https://github.com/PennyLaneAI/pennylane/pull/3874)
 
+* When using Jax-jit with gradient transforms the trainable parameters are correctly set (instead of every parameter 
```

Comment:
```suggestion
* When using `jax.jit` with gradient transforms, the trainable parameters are set correctly (instead of every parameter having
```
→ Alternatively, you could use "JAX-JIT"
→ I hope I understood right what you were trying to say in the middle of the sentence, I tried rephrasing it